packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -296,7 +296,11 @@ export class ProductAttributeValue extends PureComponent {
               block="ProductAttributeValue"
               elem="Label"
             >
-                { value }
+                { typeof value === 'string' ? (
```

Comment:
What else it can be instead of string?

Reply:
This may be a number, and then <Html content={ value } /> gives an error, because the argument should be a string